src/main/java/org/codehaus/mojo/versions/api/AbstractVersionDetails.java

```diff
@@ -243,9 +220,22 @@ public final ArtifactVersion[] getNewerVersions( String version, boolean include
         return getNewerVersions( new DefaultArtifactVersion( version ), includeSnapshots );
     }
 
+    @Deprecated
```

Comment:
one more without comments

Reply:
Added documentation with explanation for the annotation.